### PR TITLE
vision tuning angle and depth

### PR DESCRIPTION
**Pull request by NWSchallock**
I finally got to test vision!! - it works
PTAL

### CODE REVIEW

**Comment by duane701** [comment]
Closes #123

File: src/main/cpp/OI.cpp [hunk 1]
```diff
@@ -129,15 +129,15 @@ OI::OI()
 , mIsHatch(true)
 {
 
-  dStart.WhenPressed(new StageOneClimb());
+  //dStart.WhenPressed(new StageOneClimb());
```
Comment (duane701) [inline src/main/cpp/OI.cpp]: Umm, I think this is not a good idea!

File: src/main/cpp/commands/VisionDrive.cpp [hunk 2]
```diff
@@ -21,10 +21,10 @@ void VisionDrive::Initialize() {}
 // Called repeatedly when this Command is scheduled to run
 void VisionDrive::Execute() {
 
-  std::shared_ptr<OI> oi = OI::getInstance();
+  //std::shared_ptr<OI> oi = OI::getInstance();
```
Comment (duane701) [inline src/main/cpp/commands/VisionDrive.cpp]: Just delete it!

File: src/main/cpp/subsystems/Chassis.cpp [hunk 3]
```diff
@@ -33,7 +33,7 @@ Chassis::Chassis() : Subsystem(kSubsystemName),
   mLimeLightTable{nt::NetworkTableInstance::GetDefault().GetTable("limelight")},
   mIsHighGear(true),
   mHighGear(0.75),
-  mLowGear(0.50)
+  mLowGear(0.30)
```
Comment (duane701) [inline src/main/cpp/subsystems/Chassis.cpp]: Remove this change

File: src/main/cpp/subsystems/Chassis.cpp [hunk 4]
```diff
@@ -70,14 +70,29 @@ void Chassis::SetArcadeDrive(double speed, double rotation) {
...
 }
 
+double Chassis::GetVisionDepth() {
+  double pDepth = 0;
+  if (mLimeLightTable->GetNumber("tv",0.0) == 1) {
+    constexpr double maxDepthSpeed {0.5};
+    constexpr double desiredDepth {3.5};
+    double currentDepth = mLimeLightTable->GetNumber("ta",0.0);
+    double changeRequired = desiredDepth - currentDepth;
+    pDepth = (changeRequired / desiredDepth) * maxDepthSpeed;
```
Comment (duane701) [inline src/main/cpp/subsystems/Chassis.cpp]: Oh look you wrote the `P` (proportional) part of a `PID`. 

#129

**Comment by duane701** [comment]
Rebase this branch to eliminate the conflict as well.

Remap the driver buttons:

* A - climb level 1
* X - climb level 2
* Y - climb up
* B - climb down (and drive again)
* start - vision on
* back - vision

**Comment by NWSchallock** [comment]
PTAL

**Comment by NWSchallock** [comment]
replaced by #132

**Pull request closed without merging**